### PR TITLE
webpack-env: Add missing optional mode argument in require.context

### PR DESCRIPTION
**Pull request by luuksommers**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://webpack.js.org/api/module-methods/#require-context

### CODE REVIEW

File: types/webpack-env/index.d.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ declare namespace __WebpackModuleApi {
          * This creates a chunk. The chunk can be named. If a chunk with this name already exists, the dependencies are merged into that chunk and that chunk is used.
          */
         ensure(paths: string[], callback: (require: NodeRequire) => void, errorCallback?: (error: any) => void, chunkName?: string): void;
-        context(path: string, deep?: boolean, filter?: RegExp): RequireContext;
+        context(path: string, deep?: boolean, filter?: RegExp, mode?: string): RequireContext;
```
Comment (use-strict) [inline types/webpack-env/index.d.ts]:
```suggestion
        context(path: string, deep?: boolean, filter?: RegExp, mode?: "sync" | "eager" | "weak" | "lazy" | "lazy-once"): RequireContext;
```

**Review by use-strict — APPROVED** [review]
Please see my suggested changes. String is fine too.

**Comment by luuksommers** [comment]
@RyanCavanaugh @andy-ms I don't see the update published on https://www.npmjs.com/package/@types/webpack-env

**Comment by PranavSenthilnathan** [comment]
Our types publisher was having issues but it should be fixed now. An update to @types/webpack-env with your changes went out a few minutes ago. @sandersn

**Pull request merged**